packs/bestiary-effects/effect-guildmasters-lead.json

```diff
@@ -1,10 +1,10 @@
 {
     "_id": "dVVc5MYx4G3QPHoa",
-    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "img": "icons/skills/social/diplomacy-handshake.webp",
     "name": "Effect: Guildmaster's Lead",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>The member of the Copper Hand deals an additional 1d6 precision damage the first time they deal damage.</p>"
```

Comment:
Since this is an instantaneous thing we generally put those on toggles rather than effects

Reply:
Sorry, I meant on the actor itself not the effect, but I see now that this goes on a different actor so it was right before 🙃